src/benchmark_register.cc

```diff
@@ -265,7 +265,17 @@ Benchmark* Benchmark::Arg(int64_t x) {
 }
 
 Benchmark* Benchmark::Unit(TimeUnit unit) {
-  time_unit_ = unit;
+  return Unit(unit, unit);
+}
+
+Benchmark* Benchmark::CpuUnit(TimeUnit cpu_unit) {
+  time_.SetCpuUnitString(cpu_unit);
+  return this;
+}
+
+Benchmark* Benchmark::Unit(TimeUnit unit, TimeUnit cpu_unit) {
+  CpuUnit(cpu_unit);
```

Comment:
it's marginally more readable to not call `CpuUnit` here but instead call `time_.SetCpuUnitString(...)`. It saves a bit of cognitive load.

Reply:
I was trying to make it so if something special was done in CpuUnit or changes were made later, someone would've have to update 2 places, but I will change it!